test/validateProject.ts

```diff
@@ -122,13 +122,13 @@ export function getFSharpValidateOptions(projectName: string, targetFramework: s
     };
 }
 
-export function getPythonValidateOptions(projectName: string, venvName: string): IValidateProjectOptions {
+export function getPythonValidateOptions(venvName: string): IValidateProjectOptions {
     return {
         expectedSettings: {
             projectLanguage: ProjectLanguage.Python,
             projectRuntime: ProjectRuntime.v2,
-            preDeployTask: 'func: pack',
-            deploySubpath: `${projectName}.zip`,
+            deploySubpath: '.',
```

Comment:
Does this mean we aren't supporting `func: pack`?

Reply:
Again, this is related to creating a project. People can still manually change to use `func: pack` _after_ the project is created if they want to.